content/movie-review-pt2.md

```diff
@@ -0,0 +1,522 @@
+# Movie Review Program Pt2
+
+### List links for further reading here:
+
+- 
+
+# Lesson Objectives
+
+- Derive a Program Derived Address (PDA)
```

Comment:
@ZYJLiu possible to rework these into the form of
- Use X to Y
- Explain ABC
- Use M to N
- Describe PQR?

Reply:
updated